packages/blade/src/components/Amount/__tests__/mock.ts

```diff
@@ -0,0 +1,15 @@
+import type { AmountProps } from '../Amount';
+
+export const AMOUNT_SUFFIX_TEST_SET: {
+  value: number;
+  suffix: AmountProps['suffix'];
+  output: string;
+  locale?: string;
+}[] = [
+  { value: 1000000.22, suffix: 'humanize', output: '1 Mio.₹', locale: 'de-DE' },
+  { value: 1000000.0, suffix: 'decimals', output: '₹1,000,000.00', locale: 'en-US' },
+  { value: 10000000, suffix: 'none', output: '₹10,000,000', locale: 'en-US' },
+  { value: 2.07, suffix: 'decimals', output: '2,07₹', locale: 'fr-FR' },
+  { value: 2.077, suffix: 'decimals', output: '₹2.08' },
```

Comment:
See if it's possible to add a test case for: when the locale isn't set and browser locale is different. Ideally it should auto detect/fallback to browser locale. 

https://stackoverflow.com/questions/52868727/how-to-mock-window-navigator-language-using-jest

Reply:
@anuraghazra these tests are more related to i18nify's functional logic and have been covered extensively there. 
Check [here](https://github.com/razorpay/i18nify/blob/master/packages/i18nify-js/src/modules/.internal/utils/__tests__/getLocale.test.ts) for more details 

This is not required here IMO.

Reply:
Okay, should be fine if you folks have them on your end.